jamo/config/lapinpy.config

```diff
@@ -0,0 +1,44 @@
+site_name: Taskforce5 Data Management
```

Comment:
Can you remind me again what `lapinpy` is doing?

Reply:
I'm not sure I have permission to access the required repos. I am able to log into https://code.jgi.doe.gov/advanced-analysis and see a list of repos, but lapinpy doesn't appear in the list, mostly repos related to JAWS.

Reply:
LapinPy is the framework for building APIs.

Reply:
Thanks for the reminder 👍